Sources/starlight-swift/TwoPhaseSet.swift

```diff
@@ -0,0 +1,46 @@
+open class TwoPhaseSet<T> where T: Hashable {
+
+    var data: GSet<T>
+    var tomb: GSet<T>
```

Comment:
The same question, why a public getter for a public property?

Reply:
Let's make it private